internal/cli/usercmd/app.go

```diff
@@ -239,6 +239,14 @@ func (c *EpinioClient) AppShow(appName string) error {
 		}
 	}
 
+	msg = msg.WithTableRow("Routes", "")
+
+	if len(app.Configuration.Routes) > 0 {
+		for idx, route := range app.Configuration.Routes {
+			msg = msg.WithTableRow(fmt.Sprintf("  - %d", idx), route)
+		}
+	}
+
```

Comment:
This information is already printed in lines 221-226:

```
✔️  Details:
|        KEY        |                 VALUE                 |
|-------------------|---------------------------------------|
| Status            | 1/1                                   |
| Username          | 81001e8dd6dac824                      |
| StageId           |                                       |
| Age               | 43s                                   |
| Restarts          |                                     0 |
| milliCPUs         |                                     0 |
| Memory            | 3.0 MiB                               |
| Routes            |                                       |
|                   | sample2.10.86.4.38.omg.howdoi.website |
| Desired Instances |                                     1 |
| Bound Services    |                                       |
| Environment       |                                       |
| Routes            |                                       |
|   - 0             | sample2.10.86.4.38.omg.howdoi.website |

```

Reply:
@jimmykarily 

Not quite ... 221-226 shows the active routes, for when the app is running (pulled out of the Eorkload).
Only when the app is running. Nothing shown for app at rest.
My block shows the desired routes configured for the app (pulled out of the config). Even when the app is at rest.

Do wish to keep only one form ? If yes, which ?
If keeping both, we can simply fix the `KEY`s, i.e. `Active Routes`, vs `Desired Routes`.

I believe I would prefer that last.

Mixed mode: For running app show active, else show desired.

Reply:
And dd2e053605fb1f78f8d6c3491141cedb3086b223 implements mixed mode.